atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -217,8 +217,19 @@ public static LockAndTimestampServices createLockAndTimestampServices(
             Environment env,
             Supplier<RemoteLockService> lock,
             Supplier<TimestampService> time) {
+        return createLockAndTimestampServices(config, env, lock, time, () -> {
```

Comment:
If I'm not wrong this is still used by the CLIs, which are in turn used by some internal products e.g. backup tooling. I wouldn't mind - would defer to @rhero here.